docs/Makefile

```diff
@@ -0,0 +1,23 @@
+# Minimal makefile for Sphinx documentation
+#
+
+# You can set these variables from the command line, and also
+# from the environment for the first two.
+SPHINXOPTS    ?=
+SPHINXBUILD   ?= sphinx-build
+SOURCEDIR     = source
+BUILDDIR      = build
+
+# Put it first so that "make" without argument is like "make help".
+help:
+	@$(SPHINXBUILD) -M help "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
+
+.PHONY: help Makefile
```

Comment:
```suggestion
.PHONY: help livehtml docs
```

Reply:
Could you please explain?

Reply:
In phony you should put the list of the commands you define in the Makefile. That way, if there is a file in the same directory with the command name as name, the command will still be executed.
Apparently there is also a performance gain

More explanations here: https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html

> The implicit rule search (see Implicit Rules) is skipped for .PHONY targets. This is why declaring a target as .PHONY is good for performance, even if you are not worried about the actual file existing.